controllers/workoutlog.js

```diff
@@ -1,10 +1,14 @@
-const log = require('../models/workoutlog'); 
+const Log = require('../models/workoutlog');
 
 module.exports = {
-  index,
-};
+  index, 
+}; 
+
 
 function index(req, res) {
-  res.send('workoutlog/index')
+  Log.find({}, function (err, workoutlog) {
+    res.render('user/index'); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
    res.render("user/index"); 
```